lib/everypolitician/popolo/event.rb

```diff
@@ -3,6 +3,20 @@ module Popolo
     class Events < Collection; end
     class Event < Entity
       attr_accessor :start_date, :end_date
+
+      def people
+        raise NotImplementedError unless classification == 'legislative period'
```

Comment:
This approach makes me quite uncomfortable. If we're going to have different behaviour for different types of Events, I think they should be different classes.